### PR TITLE
Adds PRA full party member card

### PR DESCRIPTION
**Pull request by Ben10083**
New card variant of normal PRA member card, this was is meant for 'real' members of the Hadiist party.

PRA Consular and assistant spawns with the card

PRA ghostroles also spawn with card if applicable

image
right card is the new card

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Comment by SleepyGemmy** [comment]
> I really would like if I can make the cards spawn as registered to whoever it spawns on (consular, the ghostroles, etc), but unsure if that is possible

check the passcard/passport code. i think they spawn registered.

File: code/modules/clothing/under/accessories/xeno/tajara.dm [hunk 1]
```diff
@@ -163,6 +163,17 @@
...
+/obj/item/clothing/accessory/badge/hadii_card/member
+	name = "party member card"
+	desc = "A card denoting a member of the Hadiist party."
+	icon_state = "hadii-id-real"
+	overlay_state = "hadii-id-real"
+	badge_string = "Member of Party of the Free Tajara under the Leadership of Hadii"
+	desc_extended = "The Party of the Free Tajara under the Leadership of Hadii is the only and ruling party in the PRA, with its leader always being the elected president. \
+	They follow Hadiism as their main ideology, with the objective of securing the tajaran freedom and place in the galactic community. Membership of the Hadiist Party is not open. \
+	For anyone to become a member, they must be approved by a committee that will consider their qualifications and past. Goverment officials can grant honorary memberships, this is \
+	seen as nothing but a honor and does not grant any status or position that a regular Party member would have."
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/accessories/xeno/tajara.dm:175]:
```suggestion
	desc_extended = "The Party of the Free Tajara under the Leadership of Hadii is the only and ruling party in the PRA, with its leader always being the elected president. \
	They follow Hadiism as their main ideology, with the objective of securing the tajaran freedom and place in the galactic community. Membership of the Hadiist Party is not open. \
	For anyone to become a member, they must be approved by a committee that will consider their qualifications and past. Goverment officials can grant honorary memberships, this is \
	seen as nothing but a honor and does not grant any status or position that a regular Party member would have."
	icon_state = "hadii-id-real"
	overlay_state = "hadii-id-real"
	badge_string = "Member of Party of the Free Tajara under the Leadership of Hadii"
```

File: maps/random_ruins/exoplanets/adhomai/psis_outpost.dm [hunk 2]
```diff
@@ -56,7 +56,9 @@
 						/obj/item/melee/telebaton = 1,
 						/obj/item/handcuffs = 2
 						)
-
+	backpack_contents = list(
+						/obj/item/clothing/accessory/badge/hadii_card/member = 1
+						)
```
Comment (SleepyGemmy) [inline maps/random_ruins/exoplanets/adhomai/psis_outpost.dm:61]:
```suggestion
	backpack_contents = list(
		/obj/item/clothing/accessory/badge/hadii_card/member = 1
	)
```

File: maps/away/ships/pra/database_freighter/database_freighter_ghostroles.dm [hunk 3]
```diff
@@ -55,3 +55,4 @@
 	uniform = /obj/item/clothing/under/tajaran/database_freighter/captain
 	suit = /obj/item/clothing/suit/storage/toggle/labcoat
 	accessory = /obj/item/clothing/accessory/hadii_pin
+	l_hand = /obj/item/clothing/accessory/badge/hadii_card/member //spawns it in their hand
```
Comment (SleepyGemmy) [inline maps/away/ships/pra/database_freighter/database_freighter_ghostroles.dm:58]:
```suggestion
	l_hand = /obj/item/clothing/accessory/badge/hadii_card/member
```
self-explanatory.

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**